site/docs/extensions/copy-rows.md

```diff
@@ -64,10 +64,24 @@ This extension adds functionality for copying selected rows to the clipboard. Cu
 
 - **Detail:**
 
-   Set `true` to copy with hidden columns.
+  Set `true` to copy with hidden columns.
 
 - **Default:** `false`
 
+### copyRowsHandler
+
+- **Attribute:** `data-copy-rows-handler
+
+- **type:** `Function`
+
+- **Detail:**
+
+  Before copy rows, handle the copy rows data. The parameters object contains:
```

Comment:
Before copying rows, handle the copying rows data. The parameters object contains: